decidim-bulletin_board-app/spec/jobs/start_key_ceremony_job_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe StartKeyCeremonyJob do
+  subject { described_class.perform_now(pending_message.id) }
+
+  let!(:pending_message) { create(:pending_message, election: election, message: message) }
+  let(:election) { create(:election) }
+  let(:message) { build(:start_key_ceremony_message, election: election) }
+
+  it "processes the message" do
+    expect { subject }.to change { PendingMessage.last.status } .from("enqueued").to("accepted")
+  end
+
+  context "when the message was already processed" do
+    before { described_class.perform_now(pending_message.id) }
+
+    it "doesn't change the message status" do
+      expect { subject }.not_to change { PendingMessage.last.status } .from("accepted")
```

Comment:
Do we need the `.from("accepted")` part here as we are only interested that the status didn't get change at all?